### DISCUSSION TITLE
Need some help with lightgalleryjs

### DISCUSSION THREAD

**Discussion by Elker105** [sachinchoolur/lightGallery#1142] [General]
Hi, hope you are doing well, I need some help, I would like to know how to create a full gallery image scroller (means all images in one page) with lightgalleryjs script, like this page: https://creativemarket.com/NewTropical/5548383-The-Modern-Font-Bundle-Vol.3

Thank you very much.

**Comment by sachinchoolur** [comment]
Replied via email

**Reply by Elker105** [reply]
@sachinchoolur I didin't recieve nothing, here is my email: elkerdesign@gmail.com
thanks

**Reply by sachinchoolur** [reply]
Pls check your Twitter DM
